app/editor/src/features/content/form/CondensedContentForm.tsx

```diff
@@ -112,9 +112,11 @@ export const CondensedContentForm: React.FC<ICondensedContentFormProps> = ({
         // Only new content is initialized.
         values.contentTypeId = contentType;
         values.ownerId = userId;
-        const defaultTonePool = tonePools.find((t) => t.name === 'Default');
-        values.tonePools = !!defaultTonePool ? [{ ...defaultTonePool, value: +values.tone }] : [];
       }
+
+      const defaultTonePool = tonePools.find((t) => t.name === 'Default');
+      values.tonePools = !!defaultTonePool ? [{ ...defaultTonePool, value: +values.tone }] : [];
```

Comment:
I'm curious how this works as changing the `values.tonePools` here won't be recognized by state management because it's not part of a `useEffect`?

Reply:
Get's triggered whenever the update/create button is clicked, and the correct value will be managed in `values.tone` with `setFieldValue` from the radio buttons. So whatever value is in `values.tone` at the time of pressing the button will be added!

Reply:
and the `toModel` function is ran on the values above